src/core/src/sketch/minhash.rs

```diff
@@ -543,128 +525,95 @@ impl KmerMinHash {
         Ok((it2.count() as u64, combined_mh.mins.len() as u64))
     }
 
+    // calculate Jaccard similarity, ignoring abundance.
+    pub fn jaccard(&self, other: &KmerMinHash) -> Result<f64, Error> {
+        self.check_compatible(other)?;
+        if let Ok((common, size)) = self.intersection_size(other) {
+            Ok(common as f64 / u64::max(1, size) as f64)
+        } else {
+            Ok(0.0)
+        }
+    }
     // compare two minhashes, ignoring abundance.
+    // CTB: this is unused by ffi now.
+    // CTB: this can be replaced by a call to similarity(..., ignore_abundance=True)
```

Comment:
I'm OK with removing this on the Rust side, and update tests. Also need to bump version to `0.5.0` in `src/core/Cargo.toml` =]

Reply:
done in ed6f43a1c